### ISSUE TITLE
Validate Request Against List of Songs in Clone Hero Songs Folder

### ISSUE THREAD

**Issue by metal-messiah** [metal-messiah/JopeBot#3]

**Comment by eddiehead01** [comment]
this could possibly be easier if you were to try and validate against the songs.txt file generated when you export song list to file from CH

The hardest part is the fact that most people don't seem to write their requests in a usable format :(

**Comment by metal-messiah** [comment]
yep, i'll have to have a pretty intricate search algorithm to find "closest fit" matches on song, genre, band, charter...  i talked with Srylain and is going to start exporting songs as a .json file with more metadata

**Comment by metal-messiah** [comment]
Try to get a match “score” with icons for server. Green for high confidence, yellow for medium, red for low
